media/css/newsletter/newsletter-developer.scss

```diff
@@ -5,26 +5,40 @@
 @import '../pebbles/includes/lib';
 
 .section-intro {
-    background: url('/media/img/newsletter/developer/heart.svg') top 30px center no-repeat,
-                url('/media/img/newsletter/developer/grid.svg') top left repeat,
-                #0c99d5;
-    @include background-size(200px 199px, 60px 60px);
+    background: url('/media/img/newsletter/developer/Hearts.svg') top 30px center no-repeat,
+                url('/media/img/newsletter/developer/BG-Image.png') top left no-repeat,
+                #000000;
+
+    @include background-size(66px 44px, auto auto);
+    background-position: top 75px center, top center;
     height: 230px;
     width: 100%;
 
+    h1{
+        text-align: center;
+        color: #ffffff;
```

Comment:
Nit: can shorten to `#fff;`?

Reply:
Also, this is totally not a blocker, and not something that is really documented yet (which is definitely our bad), but we generally favor alphabeticalized CSS properties. You can probably find a plugin for your code editor that can help you with this (but like I say, not a blocker for the PR at all).

Reply:
We do have https://mozweb.readthedocs.io/en/latest/reference/css-style.html that  recommends alphabetical ordering, but a lot of that doc is in dire need of a refresh. It's a few years old and we've since changed our minds on some things. Like it says we don't use Sass but we obviously do now. Overall it's still a pretty good style guide. I should work on updating it some day...